pkgs/top-level/all-packages.nix

```diff
@@ -20574,6 +20574,8 @@ in
 
   neovim-qt = libsForQt5.callPackage ../applications/editors/neovim/qt.nix { };
 
+  glrnvim = libsForQt5.callPackage ../applications/editors/neovim/glrnvim { };
```

Comment:
```
glrnvim = callPackage ../applications/editors/neovim/glrnvim { };
```

Also it's not a QT package, so you should just call the top-level `callPackage`.